torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2077,8 +2077,20 @@ def make_arg_conj(size):
 
 
 def sample_inputs_addmm(op_info, device, dtype, requires_grad, **kwargs):
-    alpha_val = kwargs.get('alpha', 2 + 3j if dtype.is_complex else 0.6)
-    beta_val = kwargs.get('beta', 1 + 2j if dtype.is_complex else 0.2)
+    if dtype.is_complex:
+        alpha_val = 2 + 3j
+        beta_val = 1 + 2j
+    elif dtype.is_floating_point:
+        alpha_val = 2.6
+        beta_val = 3.2
+    elif not dtype.is_floating_point:
```

Comment:
This conditional is a little weird -- `if elif else` might be more natural? Or writing with more readable variable scoping

```
alpha_val = 2
beta_val = 3
if dtype.is_complex:
  ...
elif dtype.is_floating_point:
  ...
```